Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2312 -- Add `rel="noopener noreferrer"` to all external links

Resolves [EREGCSC-2312](https://jiraent.cms.gov/browse/EREGCSC-2312)

**Description:**

In order to meet MACPRO ACT (ATO) expectations, we need to ensure that all links are marked as "noopener".

This story was quite a bit more complicated than initially thought, but now every external link should (?) be properly marked with the `rel` tag and we have an end to end test for it.

**This pull request changes:**

- Adds `rel="noopener noreferrer"` to all anchor elements that have `target="_blank"`, which means they are external links (whether they have the `external` class or not)
- Updates pytest fixtures to reflect addition of `rel` tag
- Creates a Cypress utility method to check that these links have the appropriate `rel` tag and values

**Steps to manually verify this change:**

1. Click around site and inspect external links to see if they have the `rel` tag and necessary values
2. All tests pass
